docs/docs/20-usage/50-environment.md

```diff
@@ -48,94 +48,96 @@ Please note that the environment section is not able to expand environment varia
+| `CI_COMMIT_PULL_REQUEST_LABELS`  | labels assigned to pull request (set only for pull request events)                                                 | `server`                                                                                                   |
+| `CI_COMMIT_MESSAGE`              | commit message                                                                                                     | `Initial commit`                                                                                           |
+| `CI_COMMIT_AUTHOR`               | commit author username                                                                                             | `john-doe`                                                                                                 |
+| `CI_COMMIT_AUTHOR_EMAIL`         | commit author email address                                                                                        | `john-doe@example.com`                                                                                     |
+| `CI_COMMIT_AUTHOR_AVATAR`        | commit author avatar                                                                                               | `https://git.example.com/avatars/5dcbcadbce6f87f8abef`                                                     |
+| `CI_COMMIT_PRERELEASE`           | release is a pre-release (empty if event is not `release`)                                                         | `false`                                                                                                    |
+|                                  | **Current pipeline**                                                                                               |                                                                                                            |
+| `CI_PIPELINE_NUMBER`             | pipeline number                                                                                                    | `8`                                                                                                        |
+| `CI_PIPELINE_PARENT`             | number of parent pipeline                                                                                          | `0`                                                                                                        |
+| `CI_PIPELINE_EVENT`              | pipeline event (see [`event`](../20-usage/20-workflow-syntax.md#event))                                            | `push`, `pull_request`, `pull_request_closed`, `pull_request_metadata`, `tag`, `release`, `manual`, `cron` |
+| `CI_PIPELINE_EVENT_REASON`       | exact reason `pull_request_metadata` event was send. it is forge instance specific and can change                  | `label_updated`, `milestoned`, `demilestoned`, `assigned`, `edited`, ...                                   |
```

Comment:
the nice part with this is, that we can adjust per forge independently on what a forge support and if it changes it is as per documentation adjustable :)

in short terms we try to pass through what the forge reports in the webhook what event type it was ...

Reply:
... we might document this future in the forge specific doc parts ... :eye: